db/migrate/20210225163635_create_tasks.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tasks do |t|
+      t.string :description
+      t.boolean :completed
```

Comment:
You should add a default value for completed.

Reply:
Fixed whit next commit! Thanks